src/sql/src/plan/expr.rs

```diff
@@ -315,6 +322,71 @@ impl ScalarWindowFunc {
     }
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Hash, Serialize, Deserialize)]
+pub struct ValueWindowExpr {
+    pub func: ValueWindowFunc,
+    pub expr: Box<HirScalarExpr>,
+    pub order_by: Vec<ColumnOrder>,
+}
+
+impl ValueWindowExpr {
+    pub fn visit_expressions<'a, F, E>(&'a self, _f: &mut F) -> Result<(), E>
```

Comment:
I think this does want to visit `self.expr`, which it currently doesn't.

Reply:
Is this a case where I just call `f(&self.expr)` or should I do `self.expr.one_of_the_visit_methods(f)`?

Reply:
Ah, I guess [this line](https://github.com/MaterializeInc/materialize/blob/3d8f7997d957fe173dc1f072c19b5ba6153dd27e/src/sql/src/plan/expr.rs#L1659) ensures it visits them recursively, so just calling `f` seems OK.